core/lib/contract_verifier/src/etherscan/errors.rs

```diff
@@ -2,13 +2,15 @@ use zksync_dal::DalError;
 
 #[derive(Debug, thiserror::Error)]
 pub(super) enum EtherscanError {
-    #[error(transparent)]
+    #[error("Error while sending an HTTP request")]
```

Comment:
I noticed locally that the URL with the API key can be logged in some cases. That's why I've changed the error to a generic one, which should be enough for us.